src/shipping/components/ShippingMethodProducts/ShippingMethodProducts.tsx

```diff
@@ -0,0 +1,167 @@
+import Button from "@material-ui/core/Button";
+import Card from "@material-ui/core/Card";
+import IconButton from "@material-ui/core/IconButton";
+import { makeStyles } from "@material-ui/core/styles";
+import TableBody from "@material-ui/core/TableBody";
+import TableCell from "@material-ui/core/TableCell";
+import TableFooter from "@material-ui/core/TableFooter";
+import TableRow from "@material-ui/core/TableRow";
+import Typography from "@material-ui/core/Typography";
+import DeleteIcon from "@material-ui/icons/Delete";
+import CardTitle from "@saleor/components/CardTitle";
+import Checkbox from "@saleor/components/Checkbox";
+import ResponsiveTable from "@saleor/components/ResponsiveTable";
+import Skeleton from "@saleor/components/Skeleton";
+import TableCellAvatar from "@saleor/components/TableCellAvatar";
+import TableHead from "@saleor/components/TableHead";
+import TablePagination from "@saleor/components/TablePagination";
+import { renderCollection } from "@saleor/misc";
+import { ShippingZone_shippingZone_shippingMethods_excludedProducts_edges_node } from "@saleor/shipping/types/ShippingZone";
+import { ListActions, ListProps } from "@saleor/types";
+import React from "react";
+import { FormattedMessage } from "react-intl";
+
+const useStyles = makeStyles(
+  theme => ({
+    colAction: {
+      "&:last-child": {
+        paddingRight: theme.spacing(3)
+      },
+      textAlign: "right",
+      width: 100
+    },
+    colName: {
+      width: "auto"
+    },
+    colProductName: {
+      paddingLeft: 0
+    },
+    table: {
+      tableLayout: "fixed"
+    }
+  }),
+  { name: "ShippingMethodProducts" }
+);
+
+export interface ShippingMethodProductsProps
+  extends Pick<ListProps, Exclude<keyof ListProps, "onRowClick">>,
+    ListActions {
+  products: ShippingZone_shippingZone_shippingMethods_excludedProducts_edges_node[];
+  onProductAssign: () => void;
+  onProductUnassign: (ids: string[]) => void;
+}
+
+const numberOfColumns = 3;
+
+const ShippingMethodProducts: React.FC<ShippingMethodProductsProps> = props => {
+  const {
+    disabled,
+    pageInfo,
+    products,
+    onNextPage,
+    onPreviousPage,
+    onProductAssign,
+    onProductUnassign,
+    isChecked,
+    selected,
+    toggle,
+    toggleAll,
+    toolbar
+  } = props;
+
+  const classes = useStyles(props);
+
+  return (
+    <Card>
+      <CardTitle
+        title={<FormattedMessage defaultMessage="Excluded Products" />}
```

Comment:
Here might be added description with e.g. "section header" and `intl.formatMessage` instead of `FormattedMessage` for consistency.